tests/hooks/mock/seminarSeriesMock.ts

```diff
@@ -0,0 +1,84 @@
+export const mock: any[] = [
+  {
+    Name: "delectus eaque consectetur",
+    "Event Start Date": "2022-07-29",
+    "Type of Engagement": ["Seminar Series"],
+    "Event Website": "http://overlooked-sleeping.org",
```

Comment:
It might be safer to replace all mock URLs with `https://example.com/` or `https://example.org/` (a path can be appended), since it's a safe reserved URL for testing.

Relevant documents:

- https://www.rfc-editor.org/rfc/rfc2606.html
- https://www.rfc-editor.org/rfc/rfc6761.html

Reply:
Thanks for the info!